NEWS.md

```diff
@@ -1,3 +1,5 @@
+## 4.0.5
```

Comment:
Possibly new version could be 4.1.0 due to new endpoint implementation

Reply:
Done